### PR TITLE
Remove UTC label from transaction listing

### LINKED ISSUE #8386: Transaction listing pages show times in UTC.
### Describe the bug

Transactions, Disputes, and Deposits listing pages show Date / Time in `UTC` rather than in the Merchant's timezone.

This was changed in https://github.com/Automattic/woocommerce-payments/issues/7591 to align with server exports which are in UTC. While the label says `UTC` it appears to be in the Merchant's timezone. This may require a new issue to resolve.

image

### To Reproduce

1. Go to Payments > Transactions
2. See `Date / Time (UTC)`

### PR DESCRIPTION
**Pull request by brucealdridge**
Fixes #8386 

#### Changes proposed in this Pull Request

This PR removes the `(UTC)` from the `Date / Time` label on the Transactions listing page.

| Before | After |
|--------|--------|
| Before | After | 

This label was incorrectly changed in  https://github.com/Automattic/woocommerce-payments/issues/7591. While the label says UTC the time is always rendered in the Merchant's timezone.

I have created #8522 to clarify that CSV's use UTC.

#### Testing instructions

* Go to Payments > Transactions
* See `Date / Time` as column header and times in local time.

-------------------

- [x] Run `npm run changelog` 
- [x] Covered with tests 
- [x] Tested on mobile 

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `8523` or branch name `fix/8386-remove-utc-label` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=fix/8386-remove-utc-label) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: d0ecb992490a8dd5a73deaa2b90b9f8be7a5a9a6
- Build time: 2024-04-02 01:10:06 UTC

_Note: the build is updated when a new commit is pushed to this PR._

**Comment by nagpai** [comment]
✅  - Manual test checks out. I am no longer able to see `(UTC)` label. The time also seems aligned with the timezone set on the site settings. 
✅  - Did a transaction, and found  that the time is correct when settings are in UTC as well as IST ( UTC + 5:30 )

**Review by nagpai — APPROVED** [review]
Code changes are straightforward and look good to merge!

Left an optional nitpick on the changelog wording, since it would be external facing.

File: changelog/fix-8386-remove-utc-label [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Transaction listing incorrectly labeled times as UTC
```
Comment (nagpai) [inline changelog/fix-8386-remove-utc-label]: Nitpick: Is it better to reword this changelog with a verb, or the action done?  eg: 
`Remove incorrect "UTC"  label from the time column of Transactions page`

**Pull request merged**